SwiftExercises.playground/section-1.swift

```diff
@@ -36,10 +37,22 @@ Loops
 
 // Use a closed range loop to print 1 - 10, inclusively
 // WORK HERE
+for i in 1...10
+{
+    println(i)
+}
+
+
 
 // Use a half-closed range loop to print 1 - 10, inclusively
 // WORK HERE
 
+for i in 1..<10
```

Comment:
Here you actually need to change it to 1..<11 since it won't print the upper bound but the task is to print 1 - 10 inclusively